### ISSUE TITLE
[Request] Symmetrize finger poses

### ISSUE THREAD

**Issue by fluffy-critter** [rurre/PumkinsAvatarTools#71]
When I edit my hand gesture poses I often wish that I had a way of editing both hands simultaneously. It would be really nice if the pose editor had a checkbox for "keep symmetry" and maybe buttons for "transfer L->R" and "transfer R->L" or the like.

There might also be a use case for that on other body parts as well, but hand gestures in particular very often should be the same on both hands.

**Comment by rurre** [comment]
Good idea. Haven't really touched the pose editor in a long time tho so i'll see when I get back to it.
